Automattic/crowdsignal-forms
# Get rid of docker step for pot file

This PR adds `wp-cli` to the dependencies and changes the .pot creation script to use `composer exec`.

## Test instructions
Checkout and run:

  - `make install-php`
  - `NODE_ENV=production make client`
  - `make pot`

You should get a newly generated `/languages/crowdsignal-forms.pot` file without having to run the docker.
